static/app/components/replays/breadcrumbs/replayComparisonModal.tsx

```diff
@@ -25,40 +28,37 @@ export default function ReplayComparisonModal({
   replay,
   rightOffsetMs,
 }: Props) {
+  const integrationsContext = useAsyncSDKIntegrationStore();
+
   return (
     <OrganizationContext.Provider value={organization}>
-      <Header closeButton>
-        <ModalHeader>
-          <h4>
-            Hydration Error
-            <FeatureBadge type="beta" />
-          </h4>
-          <GithubFeedbackButton
-            href="https://github.com/getsentry/sentry/discussions/62097"
-            label={t('Discussion')}
-            title={null}
-            analyticsEventKey="replay.details-hydration-discussion-clicked"
-            analyticsEventName="Replay Details Hydration Discussion Clicked"
-            priority="primary"
+      <AsyncSDKIntegrationContextProvider outerContext={integrationsContext}>
```

Comment:
Because `useAsyncSDKIntegrationStore` and `<AsyncSDKIntegrationContextProvider>` are so close together here... i wonder if we need this extra step at all, it might just work without the sub-provider.

Reply:
Oh we don't, thanks for the catch. I'm a little confused cause I remembered discussing the problem with you which led to the teachings on how to use context providers... maybe the real problem was not running getsentry? 😆